### DISCUSSION TITLE
Selecting multiple devices from the device inventory list

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#86] [On hold - maybe later]
Sometimes we want to select multiple devices from some reason (for instance, for printing QR codes or other tasks). It is kind of difficult to select multiple devices. I tried to use the search window (or the column filter), then select a device, then change the search criteria to find another device and select that. But then I found out that the first selection is no longer selected. 

Can this be improved so we can run multiple search/filter actions subsequently and add some devices to an existing selection after each search action?

**Comment by YvesR** [comment]
This would result in an extra list that stores inventories, something similar like a mail distribution list. 
I doubt we can provide a feature like this. 

> As workaround it might be possible the use the training groups.
Create a training group and add an inventory (the device model) to it. Selecting the team in the inventory list will show all inventories of that device-models.
